packages/block-editor/src/components/letter-spacing-control/index.js

```diff
@@ -15,12 +15,17 @@ import useSetting from '../../components/use-setting';
 /**
  * Control for letter-spacing.
  *
- * @param {Object}   props          Component props.
- * @param {string}   props.value    Currently selected letter-spacing.
- * @param {Function} props.onChange Handles change in letter-spacing selection.
- * @return {WPElement}                      Letter-spacing control.
+ * @param {Object}   props              Component props.
+ * @param {string}   props.value        Currently selected letter-spacing.
+ * @param {Function} props.onChange     Handles change in letter-spacing selection.
+ * @param {boolean}  props.withMaxWidth Whether to restrict the field's max width.
+ * @return {WPElement}                  Letter-spacing control.
  */
-export default function LetterSpacingControl( { value, onChange } ) {
+export default function LetterSpacingControl( {
+	value,
+	onChange,
+	withMaxWidth = true,
```

Comment:
I imagine this kind of thing will be a common need and I'm not sure that exposing a new prop like `withMaxWidth` is a pattern we want to adopt widely. Would it be safer (i.e. more conservative) for now to simply expose `__unstableInputWidth` and pass that down to `UnitControl`? Then a consumer can override it with a falsy value.

```suggestion
	__unstableInputWidth = '60px',
```

Reply:
Thanks @mirka, I wasn't sure about passing through the unstable prop. I do like that approach better so have pushed the suggested change.